Documentation/ceph-cluster-crd.md

```diff
@@ -136,6 +136,7 @@ For more details on the mons and when to choose a number other than `3`, see the
   - `osdMaintenanceTimeout`: is a duration in minutes that determines how long an entire failureDomain like `region/zone/host` will be held in `noout` (in addition to the default DOWN/OUT interval) when it is draining. This is only relevant when  `managePodBudgets` is `true`. The default value is `30` minutes.
   - `manageMachineDisruptionBudgets`: if `true`, the operator will create and manage MachineDisruptionBudgets to ensure OSDs are only fenced when the cluster is healthy. Only available on OpenShift.
   - `machineDisruptionBudgetNamespace`: the namespace in which to watch the MachineDisruptionBudgets.
+- `removeOSDsIfOutAndSafeToRemove`: If `true` the operator will remove the OSDs that are out and safe-to-destroy.
```

Comment:
How about adding this? "If `true` the operator will remove the OSDs that are down and whose data has been restored to other OSDs. In Ceph terms, the osds are `out` and `safe-to-destroy` when then would be removed."